frontend/array_cmath/include/algebra/array_cmath.hpp

```diff
@@ -10,6 +10,7 @@
 // Project include(s).
 #include "algebra/math/cmath.hpp"
 #include "algebra/math/generic.hpp"
+#include "algebra/print.hpp"
```

Comment:
I don't love this. 😦

If you insist on adding print helpers, let's so that in a separate library. Something like `algebra::utils`. I really don't want this code to be always there, even when we're not interested in printing these types.

Reply:
Moved to ```algebra_utils```. The plan was to use them in detray core though, otherwise it is not really useful for logging and debugging

Reply:
`detray::core`? 😕 But `std::ostream` is not something that would ever work in GPU code. If you need logging capabilities in Detray, this is **not** the interface for doing it. 🤔

Reply:
True, but the code is fully heterogeneous, so I for myself do all of the logging (using the inspectors), as well as adhoc debugging on CPU, where the stream operators are very handy. Almost every type in detray comes with such an operator already